cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -1,11 +1,46 @@
 package cmd
 
 import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/validations"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 )
 
+func addFilenameFlag(cmd *cobra.Command, path *string) {
+	cmd.Flags().StringVarP(path, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
+}
+
+func addTinkerbellFlag(cmd *cobra.Command, path *string) {
+	cmd.Flags().StringVarP(
+		path,
+		TinkerbellHardwareCSVFlagName,
+		TinkerbellHardwareCSVFlagAlias,
+		"",
+		TinkerbellHardwareCSVFlagDescription,
+	)
+}
+
+func addManagementKubeconfigFlag(cmd *cobra.Command, path *string) {
+	cmd.Flags().StringVar(path, "kubeconfig", "", "Management cluster kubeconfig file")
+}
+
+func addBundlesOverrideFlag(cmd *cobra.Command, path *string) {
+	cmd.Flags().StringVar(path, "bundles-override", "", "Override default Bundles manifest (not recommended)")
+}
+
+func addForceCleanupFlag(cmd *cobra.Command, path *bool) {
+	cmd.Flags().BoolVar(path, "force-cleanup", false, "Force deletion of previously created bootstrap cluster")
+}
+
+func setupClusterOptionFlags(cmd *cobra.Command, clusterOpt *clusterOptions) {
```

Comment:
Think this is a natural grouping for ClusterOption flags - relevant for `create` and `upgrade`. `validate` doesn't use all the flags and `delete` has a slight modification in usage. 

I don't know if it's worth complicating the func with overrides to accommodate those commands.